docs/guides/configuration.md

```diff
@@ -1,12 +1,13 @@
 #  Merlin Configuration
 
-Before you can make serious use of Merlin, you have to setup a number of things. When using the IBM i Developer and IBM i CI/CD support within Merlin, you have to have all endpoints (IBM i, Jenkin servers) pre-defined including the credentials that are to be used. This setup process is accomplished with the following:
+Before you can make serious use of Merlin, you have to setup a number of things. When using the IBM i Developer and IBM i CI/CD support within Merlin, you have to have all endpoints (IBM i, Jenkins servers) pre-defined including the credentials that are to be used. This setup process is accomplished with the following:
 
 * **Inventory** is where you can setup connections from Merlin to IBM i, Jenkins, and other environments
 * **Credentials** your user and password and/or SSH keys belong here
 * **Templates** are used to connect a defined inventory and credentials together
 
-But of course, you have to register to your Merlin instance first.
+You can setup inventories, connections and templates by following the instructions in [Connections](./guides/platform/Manage_connections.md).
```

Comment:
You can setup inventories, connections and templates
should be 
You can setup inventories, credentials and templates